server/server.go

```diff
@@ -3430,10 +3427,14 @@ func (s *StanServer) processAck(c *channel, sub *subState, sequence uint64) {
 		return
 	}
 
-	delete(sub.acksPending, sequence)
-	stalled := sub.stalled
-	if int32(len(sub.acksPending)) < sub.MaxInFlight {
-		sub.stalled = false
+	// Do this here only if this is not a queue member. If it is, it has
```

Comment:
consider setting the `qs` variable before this `if`, then converting this to an `if/else` instead of 2 separate `if`s.